ui/components/app/name/name.tsx

```diff
@@ -21,17 +23,23 @@ export type NameProps = {
   /** Whether this is being rendered inside the NameDetails modal. */
   internal?: boolean;
 
-  /** The type of value, e.g. NameType.ETHEREUM_ADDRESS */
-  type: NameType;
-
-  /** The raw value to display the name of. */
-  value: string;
-
   /**
    * Applies to recognized contracts with no petname saved:
    * If true the contract symbol (e.g. WBTC) will be used instead of the contract name.
    */
   preferContractSymbol?: boolean;
+
+  /** The token standard, if applicable. */
+  standard?: TokenStandard;
+
+  /** Token ID, if applicable. */
+  tokenId?: Hex;
```

Comment:
Petnames is fundamentally about storing and proposing names for obscure string values, in this case Ethereum addresses.

We don't want to couple other concepts to it, rather we should be able to extract the data we need from the contract address and the chain ID alone.

Reply:
Done